db/migrate/20150817163600_deduplicate_user_identities.rb

```diff
@@ -1,7 +1,7 @@
 class DeduplicateUserIdentities < ActiveRecord::Migration
   def change
     execute 'DROP TABLE IF EXISTS tt_migration_DeduplicateUserIdentities;'
-    execute 'CREATE TEMPORARY TABLE tt_migration_DeduplicateUserIdentities AS SELECT id,provider,user_id FROM identities;'
```

Comment:
This looks like a merge from a22b6c591f2c3. @KelvinMutuma, can you rebase from master again?